src/Builder/CliMenuBuilder.php

```diff
@@ -396,22 +397,25 @@ public function setMargin(int $margin) : self
     public function setUnselectedMarker(string $marker) : self
```

Comment:
Let's just drop these methods. It's just a little BC break.

Reply:
Still needed for MenuMenuItem. I can completely drop in next PR for that item's style.

Reply:
Ok great, sounds good.